tests/kernel/timer/timer_api/testcase.yaml

```diff
@@ -7,3 +7,10 @@ tests:
     arch_exclude: riscv32 nios2 posix
     platform_exclude: qemu_x86_coverage qemu_cortex_m0
     tags: kernel userspace
+  kernel.timer.tickless.stm32:
+    filter: dt_compat_enabled("stm32,lptim")
+    platform_whitelist: nucleo_wb55rg, nucleo_l4r5zi
```

Comment:
platform white listing is not needed following check on compatible

Reply:
removing the    ` platform_whitelist: nucleo_wb55rg, nucleo_l4r5zi`